packages/runtime/test/lib/testUtils.ts

```diff
@@ -162,9 +162,7 @@ export async function syncUntilHasEvent<TEvent extends Event>(
 
 export async function uploadOwnToken(transportServices: TransportServices, forIdentity?: string): Promise<TokenDTO> {
     const response = await transportServices.tokens.createOwnToken({
-        content: {
-            content: "Hello"
-        },
+        content: { aKey: "aValue" },
```

Comment:
Mostly, we start our example values with capital letters, i.e. `"AValue"`.

Reply:
actually we don't :D

Reply:
Oops, then we'll probably do it this way or that currently 😂 Therefore, never mind for now 🌝

Reply:
@britsta better?

Reply:
Certainly better :D Even if there are still values that start with capital letters. But I don't think we need to do it all in this PR, the main thing is that the hello's and the kebab have disappeared 😂

Reply:
starting with capital a?